devicetypes/qubino/qubino-flush-2-relay.src/qubino-flush-2-relay.groovy

```diff
@@ -132,7 +132,9 @@ def excludeParameterFromSync(preference){
 def addToAssociationGroupIfNeeded() {
 	def cmds = []
 	if (zwaveInfo?.model?.equals("0052")) {
-		cmds += encap(zwave.associationV2.associationSet(groupingIdentifier: 2, nodeId: [zwaveHubNodeId]))
+		//Hub automatically adds device to multiChannelAssosciationGroup and this needs to be removed
+		cmds += encap(zwave.multiChannelAssociationV2.multiChannelAssociationRemove(groupingIdentifier: 1, nodeId:[])) 
+		cmds += encap(zwave.associationV2.associationSet(groupingIdentifier: 1, nodeId: [zwaveHubNodeId]))
```

Comment:
This _should_ be done automatically. Were you finding that only the multi-channel association was being set and not the regular association?

Reply:
Yes, only multi-channel was set and this causes issues